unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -340,19 +340,29 @@ protected void HideAllAgentRenderers()
             HashSet<Vector3> seenPoints = new HashSet<Vector3>();
             int layerMask = 1 << 8;
             int stepsTaken = 0;
+
+            //Traverse entire room in every direction with every step, stopping at any points that repeat and continuing on from ones that don't yet
```

Comment:
For each of the comments, can you add a single space after the backslashes //?

So this goes from:
```c#
//Traverse entire room in every direction...
```
to:
```c#
// Traverse entire room in every direction...
```
This is a C# style convention and makes things look a bit less cluttered.

Reply:
Done